src/main/java/de/tum/in/www1/artemis/service/ComplaintResponseService.java

```diff
@@ -258,16 +258,16 @@ public boolean isUserAuthorizedToRespondToComplaint(Complaint complaint, User us
             return false;
         }
         // for teams: the tutor who is responsible for team, should evaluate the complaint
-        if (participation.getParticipant() instanceof Team) {
-            return assessor.getLogin().equals(user.getLogin());
+        if (participation.getParticipant() instanceof Team team) {
+            return user.getLogin().equals(team.getOwner().getLogin());
```

Comment:
I am not quite sure if `user` is a well-chosen name here. We already know that the user is at least a tutor, maybe the code readability would increase if this variable was named `tutor` / `teachingAssistant` instead _(I would prefer `teachingAssistant`, this is used interchangeably with `tutor` in many places, but for me it is more general than `tutor`)_.

Reply:
You‘re right. I didn’t refactor the name of the parameter.
One more thing for the followup